src/types/module.rs

```diff
@@ -127,9 +127,11 @@ impl PyModule {
 
             let mptr = ffi::PyImport_ExecCodeModuleEx(module.as_ptr(), cptr, filename.as_ptr());
             if mptr.is_null() {
+                ffi::Py_DECREF(cptr);
                 return Err(PyErr::api_call_failed(py));
             }
 
+            ffi::Py_DECREF(cptr);
```

Comment:
I think now it's just equivalent to this?

```suggestion
            let mptr = ffi::PyImport_ExecCodeModuleEx(module.as_ptr(), cptr, filename.as_ptr());
            ffi::Py_DECREF(cptr);
            if mptr.is_null() {
                return Err(PyErr::api_call_failed(py));
            }

```